.template/addons/svgeez/.github/wiki/template.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+if Dir.exist?('.github/wiki')
+  use_source_path __dir__
+
+  copy_file 'Add-new-svg-icon.md', '.github/wiki/Add-new-svg-icon.md'
```

Comment:
As we now have renamed the section,  let's  rename the file as well 🙌🏿 

```suggestion
  copy_file 'Managing-SVG-Icons.md', '.github/wiki/Managing-SVG-Icons.md'
```

Reply:
Thanks. Fixed in b21c85ca605d162ece23e0bbcc2a89f9d1eb8457